bin/do_username

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env do_username
+
+require 'do_username'
+if ARGV.length < 1
+    puts DOUsername.generate()
```

Comment:
You don't need the empty parenthesis `()` when the method call doesn't have arguments ;)

```suggestion
  puts DOUsername.generate
```

Reply:
Coming from JS, that's new to me 😮